Dec2022Demo/test_S1.jl

```diff
@@ -0,0 +1,116 @@
+types′ = LabelledPetriNet([:Pop],
+    :infect=>((:Pop, :Pop)=>(:Pop, :Pop)),
+    :disease=>(:Pop=>:Pop),
+    :strata=>(:Pop=>:Pop))
+types = map(types′, Name=name->nothing)
+
+
+SIRD = LabelledPetriNet([:S, :I, :R, :D],
+    :inf => ((:S,:I) => (:I,:I)),
+    :recover => (:I=>:R),
+    :death => (:I=>:D));SIRD = LabelledPetriNet([:S, :I, :R, :D],
+    :inf => ((:S,:I) => (:I,:I)),
+    :recover => (:I=>:R),
+    :death => (:I=>:D));
+AlgebraicPetri.Graph(SIRD)
+
+SIRD_typed = homomorphism(SIRD, types;
+initial=(T=[1,2,2],I=[1,2,3,3],O=[1,2,3,3]),
+type_components=(Name=x->nothing,))
+@assert is_natural(SIRD_typed)
+
+
+p_i(u,t) = 0.000025-0.000025*cos(t/100*2*pi)
```

Comment:
What are these doing? Why do we have cosine? In general, for these demos it would be good even if these aren't in notebooks to have a "literate" code mentality, where you write out the mathematics/semantics before the code in order to "preload" the work for the presentation.